docs/content/connectors/db2-cdc.md

```diff
@@ -246,8 +246,7 @@ Data Type Mapping
 <table class="colwidths-auto docutils">
     <thead>
       <tr>
-        <th class="text-left" style="width:30%;">Db2 type<a href="https://dev.mysql.com/doc/man/8.0/en/data-types.
-html"></a></th>
+        <th class="text-left" style="width:30%;">Db2 type<a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types"></a></th>
```

Comment:
Do users need to visit the link here？if necessary,Do you want to change it to the following form?

<th class="text-left" style="width:30%;"><a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types">Db2 type</a></th> 

The current writing method does not seem to generate any hyperlink.

Reply:
Thanks @rookiegao to point this.
Fixed.